core/uvisor-config.h

```diff
@@ -40,7 +40,7 @@
  * is configuration-specific. See \ref UVISOR_SRAM_LENGTH_USED for more
  * information.
  */
-#define UVISOR_SRAM_LENGTH_PROTECTED 0x2000
```

Comment:
This was 8 KiB. It's now 16 KiB? Is that really necessary?

Reply:
Actually it needs to be 0x3000, but then fail on:
#if (SRAM_ORIGIN % UVISOR_SRAM_LENGTH_PROTECTED) != 0
#error "The SRAM origin must be aligned to a multiple of UVISOR_SRAM_LENGTH_PROTECTED"
#endif

Reply:
Can we align it properly? Or is the assertion incorrect?